### PR TITLE
Finalize Request #8216

### PR DESCRIPTION
**Pull request by unl-creq**
Updating bulletin section.

### CODE REVIEW

**Comment by tlemburg** [comment]
Looks good

File: data/2015/majors/Pre-Veterinary Medicine.xhtml [hunk 1]
```diff
@@ -14,56 +14,67 @@
...
+<p class="title-1">PPVM Pre-Veterinary Medicine Core</p>
+
+<p class="MsoNormal" style="margin: 3.9pt -1pt 0.0001pt 4.5pt; line-height: 11.3pt;"><span style="font-family: Verdana, sans-serif;"><span style="letter-spacing: 0.1pt;"></span><span style="letter-spacing: -0.05pt;"></span><span style="letter-spacing: -0.05pt;"></span><span style="letter-spacing: 0.15pt;"></span><span style="letter-spacing: -0.05pt;"></span><span style="letter-spacing: -0.5pt;"></span><span style="letter-spacing: 0.15pt;"></span><span style="letter-spacing: -0.05pt;"></span><span style="letter-spacing: 0.15pt;"></span><span style="letter-spacing: -0.05pt;"></span><span style="letter-spacing: 0.15pt;"></span><span style="letter-spacing: 0.15pt;"></span><span style="letter-spacing: 0.1pt;"></span><span style="letter-spacing: -0.95pt;"></span><span style="letter-spacing: 0.05pt;"></span><span style="letter-spacing: 0.1pt;"></span><span style="letter-spacing: -0.05pt;"></span></span></p>
```
Comment (kabel) [inline data/2015/majors/Pre-Veterinary Medicine.xhtml]: Need to remove this empty element.

File: data/2015/majors/Pre-Veterinary Medicine.xhtml [hunk 2]
```diff
@@ -14,56 +14,67 @@
...
+<p><span style="line-height: 11.3pt; font-family: Verdana, sans-serif;"></span><span style="color: #333333; font-family: Verdana, sans-serif;"><span style="line-height: 15.0666666030884px;"></span></span><span style="line-height: 11.3pt; font-family: Verdana, sans-serif;"></span><span style="line-height: 11.3pt; font-family: Verdana, sans-serif;"></span><span style="line-height: 11.3pt; font-family: Verdana, sans-serif;"></span><span style="line-height: 11.3pt; font-family: Verdana, sans-serif;"></span><span style="line-height: 11.3pt; font-family: Verdana, sans-serif;"> </span></p>
```
Comment (kabel) [inline data/2015/majors/Pre-Veterinary Medicine.xhtml]: Need to remove this empty element.

File: data/2015/majors/Pre-Veterinary Medicine.xhtml [hunk 3]
```diff
@@ -71,20 +82,29 @@
...
-<p class="requirement-sec-1">Economics, Humanities and Social Sciences 15</p>
-<p class="requirement-sec-2">ECON 200 or ECON 211 <span class="requirement-bold">or</span> ECON 212 <span class="requirement-bold">or</span> AECN 141 <span class="requirement-ital">(ACE 6)</span> 3</p>
-<p class="requirement-sec-2">ACE Courses 12</p>
+<p class="requirement-sec-1">Economics ACE 6 3</p>
+<p class="requirement-sec-2 requirement-ital">Select one from:</p>
+<p class="requirement-sec-3">AECN 141 Introduction to the Economics of Agriculture (3 cr)</p>
+
+<p class="requirement-sec-3">ECON 200 Economic Essentials and Issues (3 cr)</p>
+<p class="requirement-sec-3">ECON 211/ECON 211H Principles of Macroeconomics (3 cr)</p>
+<p class="requirement-sec-2"><span class="requirement-sec-3"></span> </p>
```
Comment (kabel) [inline data/2015/majors/Pre-Veterinary Medicine.xhtml]: Remove this empty element.

**Pull request merged**